_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+
+[2011 AG report](https://www.oag-bvg.gc.ca/internet/English/parl_oag_201106_02_e_35370.html): Examination of the progress made from the 2006 recommendations
+
+Here are some of the findings:
+
+- Overall, the government has made unsatisfactory progress on its commitments in response to our 2006 recommendations.Although some improvements have been made, progress has been unsatisfactory in the important areas of governance and project business cases. Only two of the five projects we looked at, the Temporary Resident Biometrics Project and Integrated Revenue Collections systems, met most of our criteria for well-managed projects.
+- In order to increase the likelihood of success, departments have significantly reduced the scope of the projects and considerably extended their timelines.In the area of project governance, the Expenditure Management Information System and Global Case Management System have had important deliverables deferred without full analysis of the impacts and costs of not completing these projects. In three of the five projects examined, the project business cases did not identify measurable benefits or the benefits have not been measured.For example, the 2007 business case for the Secure Channel no longer quantifies the financial benefits.
+
+Unfortunately, after much effort, these decade old findings are still prevalent today. The underlying issue is slow response to change, and a large lead time to be able to incorporate them into projects and systems. One of the culprits of this is a complex release process which tends to promote large releases due to the efforts needed to go through it. [Large releases are risky](https://sencanada.ca/content/sen/committee/421/NFFN/reports/NFFN_Phoenix_Report_32_WEB_e.pdf). Each release involves many different stakeholders with distinct functional roles, and manual checks and handoffs. If you're looking at a diagram of this process for the first time, it's almost impossible to comprehend. It is difficult for one person to completely describe all the stages that it involves.
+
+### Putting the release process into perspective
+
+Here are some statistics to help you understand the scale of the complexity. These come from the work of the Senior Advisors (SA) have [mapped](http://dialogue/grp/PR6893344/OneNote/AppDevSA/02-Development%20%28In%20Progress%29/Outcomes%20Map.one#Feature%20to%20Release%20BPM&section-id=%7B5E16E60C-310B-49EF-8451-88E0CE4DA968%7D&page-id=%7B8208D63D-CE7D-49EB-BF48-033A38FA5825%7D&end) ( this is an internal link ) the current release process here at ESDC as a means to identify improvements. Assume you are a developer, you are tasked with adding 4 fields to a form on a front-facing application, this is about 2 weeks of actual development work. You have to alter some front end code, add input validation in your views and perhaps alter a database schema to capture these fields. With our current process:
```

Comment:
How about 'Assume you are a developer tasked with adding 4 fields to a form on a front-facing application.  To do this task you have to alter some front end code, add input validation in your views and perhaps alter a database schema to capture these fields.  This is about 2 weeks of actual development work.  With our current process:.....